packages/frontend/src/components/RichTextEditor/SuggestionPopper.tsx

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react'
+import { ExpandLess, ExpandMore } from '@mui/icons-material'
+import {
+  Button,
+  Collapse,
+  List,
+  ListItemButton,
+  ListItemText,
+  Paper,
+  Popper,
+  Typography,
+} from '@mui/material'
```

Comment:
Eh, since we're refactoring anyway, can we get rid of this popper? Kill the old material code, use chakra only

Reply:
Nvm I see that you are removing this in #278